packages/angular/src/stories/form/form.stories.ts

```diff
@@ -29,105 +33,196 @@ export default {
 };
 
 export const PasswordDemo = () => {
+  const model = {
+    radio: '',
+  };
+  const required = boolean('Required form controls', true);
+  const controlHelper = boolean('Helper Text', true);
+  const disabled = boolean('Disable Password', false);
   return {
     name: 'Password',
     template: passwordTemplate.default,
+    props: {
+      model,
+      required,
+      controlHelper,
+      disabled,
+      onSubmit: formSubmit,
+    },
   };
 };
 
 export const RadioDemo = () => {
+  const model = {
+    options: '',
+  };
+  const required = boolean('Required form controls', true);
+  const controlHelper = boolean('Helper Text', true);
+  const disabled = boolean('Disable Radio', false);
+  const inline = boolean('Inline layout', false);
   return {
-    name: 'Radio',
+    name: 'Password',
     template: radioTemplate.default,
+    props: {
+      model,
+      required,
+      controlHelper,
+      disabled,
+      inline,
+      onSubmit: formSubmit,
+    },
   };
 };
 
 export const RangeDemo = () => {
+  const rangeValue = 50;
+  const controlHelper = boolean('Helper Text', true);
+  const disabled = boolean('Disable Range', false);
   return {
     name: 'Range',
     template: rangeTemplate.default,
+    props: {
+      rangeValue,
+      controlHelper,
+      disabled,
+      onSubmit: formSubmit,
+      change: event => {
+        action('Range value change')(event.target.value);
+      },
+    },
   };
 };
 
 export const SelectDemo = () => {
+  const required = boolean('Required form controls', true);
+  const disabled = boolean('Disable Select', false);
+  const helpers = boolean('Helper Text', true);
+  const errors = boolean('Error Text', true);
   return {
     name: 'Select',
     template: selectTemplate.default,
+    props: {
+      required,
+      disabled,
+      helpers,
+      errors,
+      onSubmit: formSubmit,
+    },
   };
 };
 
 export const TextareaDemo = () => {
+  const required = boolean('Required form controls', true);
+  const disabled = boolean('Disable Textarea', false);
+  const helpers = boolean('Helper Text', true);
+  const errors = boolean('Error Text', true);
   return {
     name: 'Textarea',
     template: textareaTemplate.default,
+    props: {
+      required,
+      disabled,
+      helpers,
+      errors,
+      onSubmit: formSubmit,
+    },
   };
 };
 
 export const ToggleDemo = () => {
+  const model = {
+    option1: true,
+  };
+  const required = boolean('Required form controls', true);
+  const disabled = boolean('Disable Toggles', false);
+  const helpers = boolean('Helper Text', true);
+  const errors = boolean('Error Text', true);
   return {
     name: 'Toggle',
     template: toggleTemplate.default,
+    props: {
+      required,
+      disabled,
+      helpers,
+      errors,
+      onSubmit: formSubmit,
+    },
   };
 };
 
-export const FullDemo = () => {
-  return {
-    name: 'Full Demo',
-    template: fullTemplate.default,
-  };
-};
+// export const FullDemo = () => {
```

Comment:
Do you still need to have it here?

Reply:
I'm going to leave it for now. I hope to have time to put together a full demo with all controls for forms.